bin/uwsgi.ini

```diff
@@ -7,7 +7,7 @@ module = nucleus.wsgi:application
 
 die-on-term = true  ; Shutdown on SIGTERM (default is respawn)
 enable-threads = true  ; Support multithreading
-harakiri = 30  ; Kill hung requests after 30 seconds
+harakiri = 300  ; Kill hung requests after 30 seconds
```

Comment:
I hate to be pedantic, but would you mind also updating the comment?

Reply:
Also, is 5min too much? Would it be better to just bump it to 60 or 90s first?

Reply:
sure